internal/cli/auth/login.go

```diff
@@ -110,10 +117,67 @@ Your code will expire after %.0f minutes.
 	opts.AuthToken = accessToken.AccessToken
 	opts.RefreshToken = accessToken.RefreshToken
 	opts.SetUpAccess()
+	_, _ = fmt.Fprintf(opts.OutWriter, "Successfully logged in.\n")
+	if opts.noProfile {
+		return opts.config.Save()
+	}
+	if err := opts.InitStore(ctx); err != nil {
+		return err
+	}
+	_, _ = fmt.Fprintf(opts.OutWriter, "Press Enter to continue your profile configuration")
+	_, _ = fmt.Scanln()
+	if err := opts.askOrg(); err != nil {
+		return err
+	}
+	if err := opts.askProject(); err != nil {
+		return err
+	}
+
+	opts.SetUpProject()
+	opts.SetUpOrg()
+	if err := survey.Ask(opts.DefaultQuestions(), opts); err != nil {
+		return err
+	}
+	opts.SetUpOutput()
+	opts.SetUpMongoSHPath()
 	if err := opts.config.Save(); err != nil {
 		return err
 	}
-	_, _ = fmt.Fprintf(opts.OutWriter, "Successfully logged in\n")
+	_, _ = fmt.Fprintf(opts.OutWriter, "\nYour profile is now configured.\n")
+	if config.Name() != config.DefaultProfile {
+		_, _ = fmt.Fprintf(opts.OutWriter, "To use this profile, you must set the flag [-%s %s] for every command.\n", flag.ProfileShort, config.Name())
+	}
+	_, _ = fmt.Fprintf(opts.OutWriter, "You can use [%s config set] to change these settings at a later time.\n", config.ToolName)
+	return nil
+}
+
+func (opts *loginOpts) askOrg() error {
+	oMap, oSlice, err := opts.Orgs()
+	var orgID string
+	if err != nil || len(oSlice) == 0 {
+		return errors.New("no orgs")
+	}
+
+	p := prompt.NewOrgSelect(oSlice)
+	if err := survey.AskOne(p, &orgID); err != nil {
+		return err
+	}
+	opts.OrgID = oMap[orgID]
+	return nil
+}
+
+func (opts *loginOpts) askProject() error {
+	pMap, pSlice, err := opts.Projects()
+	var projectID string
+	if err != nil || len(pSlice) == 0 {
+		return errors.New("no projects")
+	}
+
+	p := prompt.NewProjectSelect(pSlice)
+	if err := survey.AskOne(p, &projectID); err != nil {
+		return err
+	}
+	opts.ProjectID = pMap[projectID]
```

Comment:
most of the code here can be found also in [config command](https://github.com/mongodb/mongocli/blob/master/internal/cli/config/config.go)
For consistency (especially for the future, if we'll change any of these 2 commands) is it possible to share the common logic (I'd say config.go seems like a more suitable place), or we're trying to keep the commands as isolated as possible?

Reply:
there's a main difference is that for config we have a fallback case where we don't do a select but let users input the id directly, this happens if they don't provide API keys which is allowed and would not happen for login, the main part that's shared `opts.Projects()` is being extracted as par of this PR